### ISSUE TITLE
Blink sometimes shows an empty bar on the bottom at startup with an external keyboard connected

### ISSUE THREAD

**Issue by kkk669** [blinksh/blink#1125]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

- Blink v13.5.7.244
- iPadOS 14.2 (18B5052h)

### Describe the bug

I don't know what conditions cause this bug, but Blink sometimes shows an empty bar on the bottom at startup with an external keyboard connected. Maybe, it seems to happen after leaving Blink in the background for a while.

And, if I tap anywhere in this state, the bar will disappear.

![AE85E8BE-29BA-4494-9CA0-A0856AD06B86](https://user-images.githubusercontent.com/601636/94360363-9dcf8380-00e7-11eb-9dfd-37666f469623.jpeg)

**Comment by gyre** [comment]
I have this too.  I tried disabling keyboard shortcuts under settings / general / keyboard / shortcuts a few minutes ago and it hasn't re-appeared... yet.

**Comment by gyre** [comment]
Of course, that means I can't use my password manager 1password anymore :)  So not too useful.

**Comment by kkk669** [comment]
I've found steps to reproduce the issue.

1. Close all windows of Blink Shell on the App Switcher
1. Connect an external keyboard, e.g. Magic Keyboard, into an iPad
1. Open Blink Shell
1. Open another window in Split View
1. Type something

![96C12DC8-E100-47E8-ADB1-4F28E411A2F4](https://user-images.githubusercontent.com/601636/99904293-5e9b6880-2d0d-11eb-9b3d-c061c302c810.gif)

**Comment by kkk669** [comment]
It seems that my PR didn't completely fix the problem. It still occurs.

**Comment by kkk669** [comment]
@yury I'm sorry, but could you please reopen this issue? Thanks.

**Comment by kkk669** [comment]
I found an 100% reproducible steps. (iPadOS 14.4.2)

1. Force quit all Blink Shell's windows
2. Open Safari
3. Open Blink Shell

Then, an empty input accessory view will be shown on the bottom of the screen.

**Comment by gyre** [comment]
You may have to press a key before the blank bar appears.  At least for me.

**Comment by kkk669** [comment]
@gyre What iOS version are you using?

**Comment by gyre** [comment]
ipados 14.4.2.  but it's been doing this for all the 14.* releases.

**Comment by kkk669** [comment]
@gyre Thank you for the information. If possible, could you test #1220? In my environment, it fixes the issue.

**Comment by gyre** [comment]
I don't own a mac, so I'd presumably have to wait until a binary hits TestFlight.  But thanks in advance for looking into this.

**Comment by gyre** [comment]
looks like it's hit TF.  so far, so good.  thanks!!! :)

**Comment by kkk669** [comment]
No, that is 77f43a2770861ed9ca7bc5df27396d71b71725fd. So, my workaround is no longer necessary.
